package.json

```diff
@@ -43,10 +43,12 @@
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-plugin-import": "^2.18.2",
     "express": "^4.17.1",
+    "got": "^9.6.0",
+    "lodash.difference": "^4.5.0",
     "lodash.uniq": "^4.5.0",
+    "mkdirp": "^0.5.1",
```

Comment:
u can pass `recursive: true` to `fs.mkdir` in nodejs 10

Reply:
won't work, needs `-p`